<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>780</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>lifemotif-desktop</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="LifeMotifCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="diaryList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="diaryListLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>일기 목록</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mimeRawMessageButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>일기 원본 MIME 메시지 보기...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="attatchmentGroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>221</width>
      <height>93</height>
     </rect>
    </property>
    <property name="title">
     <string>첨부파일</string>
    </property>
    <widget class="QPushButton" name="attatchmentSaveAsButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>201</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>첨부파일 저장...</string>
     </property>
    </widget>
    <widget class="QComboBox" name="attatchmentComboBox">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>30</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="diaryHtmlText">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>751</width>
      <height>311</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="toEdit">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>40</y>
      <width>691</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fromEdit">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>12</y>
      <width>691</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="subjectEdit">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>70</y>
      <width>691</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="diaryPlainText">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>130</y>
      <width>751</width>
      <height>261</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="toLabel">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>40</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>받는 사람: </string>
    </property>
   </widget>
   <widget class="QLabel" name="fromLabel">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>12</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>보낸 사람: </string>
    </property>
   </widget>
   <widget class="QLabel" name="subjectLabel">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>70</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>제목:        </string>
    </property>
   </widget>
   <widget class="QPushButton" name="mimeStructureButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>일기 MIME 구조 보기...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearTextButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>화면 지우기</string>
    </property>
   </widget>
   <widget class="QLabel" name="plainTextLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Plain Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="htmlTextLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>410</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HTML Text</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showHtmlCodeCheckBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>410</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>HTML 태그를 보이도록</string>
    </property>
   </widget>
   <zorder>calendarWidget</zorder>
   <zorder>diaryList</zorder>
   <zorder>diaryListLabel</zorder>
   <zorder>mimeRawMessageButton</zorder>
   <zorder>attatchmentGroupBox</zorder>
   <zorder>diaryHtmlText</zorder>
   <zorder>toEdit</zorder>
   <zorder>fromEdit</zorder>
   <zorder>subjectEdit</zorder>
   <zorder>diaryPlainText</zorder>
   <zorder>toLabel</zorder>
   <zorder>fromLabel</zorder>
   <zorder>subjectLabel</zorder>
   <zorder>mimeStructureButton</zorder>
   <zorder>clearTextButton</zorder>
   <zorder>plainTextLabel</zorder>
   <zorder>htmlTextLabel</zorder>
   <zorder>showHtmlCodeCheckBox</zorder>
   <zorder>attatchmentComboBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTestMenu">
    <property name="title">
     <string>LifeMotif</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionBrowserAuthentication"/>
    <addaction name="separator"/>
    <addaction name="revokeAuthentication"/>
    <addaction name="separator"/>
    <addaction name="actionListPreference"/>
    <addaction name="actionBuildLocalStructure"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionPreference"/>
   </widget>
   <addaction name="menuTestMenu"/>
   <addaction name="menuOption"/>
  </widget>
  <action name="actionConsoleAuthentication">
   <property name="text">
    <string>콘솔에서 인증하기...</string>
   </property>
  </action>
  <action name="actionListPreference">
   <property name="text">
    <string>Preference</string>
   </property>
  </action>
  <action name="actionBuildLocalStructure">
   <property name="text">
    <string>이메일 구조 가져오기</string>
   </property>
  </action>
  <action name="actionBrowserAuthentication">
   <property name="text">
    <string>인증하기...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>이 프로그램은...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>종료</string>
   </property>
  </action>
  <action name="revokeAuthentication">
   <property name="text">
    <string>인증 취소하기...</string>
   </property>
  </action>
  <action name="actionPreference">
   <property name="text">
    <string>Preference...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LifeMotifCalendarWidget</class>
   <extends>QCalendarWidget</extends>
   <header>lifemotif_calendar_widget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
